live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/hx_insight/storage/connector/FileUploader.java

```diff
@@ -27,5 +27,8 @@
 
 public interface FileUploader
 {
+    @SuppressWarnings({"PMD.CloseResource", "PMD.PreserveStackTrace"})
```

Comment:
I don't think we need this.

Reply:
done in [260c2e1](https://github.com/Alfresco/hxinsight-connector/pull/274/commits/260c2e120a8371465126244b8ca2362e94ece209)